TLM/TLM/Resources/lang_pl.txt

```diff
@@ -230,5 +230,6 @@ Vehicles_may_turn_on_red Pojazdy mogą skręcić w prawo na czerwonym świetle
 Also_apply_to_left/right_turns_between_one-way_streets Uwzględnia również skręt w lewo/prawo pomiędzy drogami jednokierunkowymi
 Scan_for_known_incompatible_mods_on_startup Uruchamiaj sprawdzanie przy starcie gry
 Ignore_disabled_mods Ignoruj wyłączone mody
-Traffic_Manager_detected_incompatible_mods Traffic Manager wykrył niekompatybilne mody
-Notify_me_if_there_is_an_unexpected_mod_conflict Powiadom mnie w razie nieoczekiwanej niezgodność modów
\ No newline at end of file
+Traffic_Manager_detected_incompatible_mods wykrył niekompatybilne mody
+Notify_me_if_there_is_an_unexpected_mod_conflict Powiadom mnie w razie nieoczekiwanej niezgodność modów
+Unsubscribe Unsubscribe
```

Comment:
`Unsubscribe Odsubskrybuj` but better would be `Unsubscribe Anuluj subskrypcję` but we don't have that much space

Reply:
Updated with `Unsubscribe Odsubskrybuj`